IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/Entities/IfcKerb/DocEntity.xml

```diff
@@ -3,7 +3,7 @@
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="17ec7cf6-475d-43b2-ba7e-f765ae319696" DefinedType="IfcKerbTypeEnum" AttributeFlags="1">
 			<Documentation>Identifies the predefined type of a kerb element. This type may associate additional specific property sets.
->NOTE  The PredefinedType shall only be used, if no IfcKerbType is assigned, providing its own IfcKerbType.PredefinedType.</Documentation>
+&gt; NOTE  The PredefinedType shall only be used, if no IfcKerbType is assigned, providing its own IfcKerbType.PredefinedType.</Documentation>
 		</DocAttribute>
 	</Attributes>
 	<WhereRules>
```

Comment:
This change seems to be present in more than one PR.

Reply:
Indeed, but no worries. It is already in the develop branch so just ignore.